gomint-api/src/main/java/io/gomint/entity/EntityPlayer.java

```diff
@@ -319,4 +320,40 @@
      */
     boolean getFlying();
 
+    /**
+     *
+     * Send a title text to the user's screen, with an optional subtitle.
+     *
+     * @param title
+     * @param subtitle
+     * @param fadein
+     * @param duration
+     * @param fadeout
+     * @param unit
+     */
+
+    void sendTitle( String title, String subtitle, long fadein, long duration, long fadeout, TimeUnit unit );
+
+    /**
+     * Send a title without subtitle.
+     *
+     * @param title
+     *
+     * Default time for fadin and duration is 1 second.
+     */
+
+    void sendTitle( String title );
+
+    /**
+     *
+     * Send a title with title and subtitle.
+     *
+     * @param title
```

Comment:
@param title Big text displayed in the middle of the screen
@param Subtitle Smaller big text displayed below the title text